### DISCUSSION TITLE
Thank you Tailwind Variants!

### DISCUSSION THREAD

**Discussion by 2khan** [heroui-inc/tailwind-variants#28] [General]
Hello!

I am a Senior Frontend Developer from Mongolia and I've been working on an auction/ecommerce web app at a startup. My main focus is reducing the bundle size while moving fast (changing designs after user tests). The whole project is built using `React` and `tailwindcss` and I've built my own design system integration that kept scaling as the app grew bigger. I even changed from tailwind to Stitches because their API was great. Unfortunately Stitches is a runtime solution and didn't meet our criteria on bundle sizes and performance. Then I discovered Tailwind Variants. Now I can implement my design system with a great API without compromising bundle size.

**I'd like to thank the team** for creating and maintaining such a **valuable** project. I know there will be more developers like me who will appreciate the work that you're doing. Keep rocking, team!

**Comment by jrgarciadev** [comment]
Hello @thejskhan huge thanks 🙏🏻, it motives us to keep pushing, I'm glad this project is helping you 🚀
